hhub/views.py

```diff
@@ -108,3 +121,24 @@ def search_entries(request):
 
 def entry_platform(request, platform):
     return
+
+#########
+# UTILS #
+#########
+def sort_and_order(entries, order_by, sort_by):
+    try:
+        # order_by a specific field from schema
+        if order_by != 1 and order_by != "":
+            # specify an order --> asc is by default, so we're not dealing with it
+            if sort_by != 1 and sort_by != "" and sort_by.lower() != "asc":
```

Comment:
If we already inside `if order_by != 1 and order_by != "":`, why is the same condition repeated inside?